conda_mirror/conda_mirror.py

```diff
@@ -273,29 +355,56 @@ def _download(url, target_directory, package_metadata=None, validate=True,
 
 
 def _list_conda_packages(local_dir):
+    """List the conda packages (*.tar.bz2 files) in `local_dir
+
+    Parameters
+    ----------
+    local_dir : str
+        Some local directory with (hopefully) some conda packages in it
+
+    Returns
+    -------
+    list
+        List of conda packages in `local_dir`
+    """
     contents = os.listdir(local_dir)
     return fnmatch.filter(contents, "*.tar.bz2")
 
 
-def _validate_packages(repodata, package_directory):
+def _validate_packages(package_repodata, package_directory):
+    """Validate local conda packages.
+
+    NOTE: This is slow.
+    NOTE2: This will remove any packages that are in `package_directory` that
```

Comment:
It will also remove any packages that fail validation because that's what `_validate` does. Maybe worth mentioning?